mobile/lib/common/status_screen.dart

```diff
@@ -67,25 +65,26 @@ class _StatusScreenState extends State<StatusScreen> {
               ),
             ],
           )),
+      Visibility(
```

Comment:
This is cool, I didn't know about this widget :)

Reply:
yeah, saves painful array constructs - we've used it in a couple of places in the past

Reply:
Makes sense!